core/templates/pages/contributor-dashboard-page/modal-templates/question-suggestion-review.component.html

```diff
@@ -185,6 +185,11 @@ <h3 class="oppia-mobile-question-header" title="{{questionHeader}}">
     margin-left: 10px;
     margin-top: 20px;
   }
+  .fa, .fas {
```

Comment:
I'm concerned about this, we shouldn't override .fa and .fas as they are general classes. If you need to customize the CSS, please use an oppia-specific class instead, like the others in this file.

Thanks!